### ISSUE TITLE
tk --init=<type> is not working

### ISSUE THREAD

**Issue by nandorholozsnyak** [linux-china/task-keeper#1]
Hi.

I just wanted to try out the `--init` command like it is mentioned in the release notes for 0.5.0 and 0.5.2, but with the given examples it is not working.

```
❯ tk --init=jbang
error: The argument '--init <shell> <make> <jbang> <just>' requires 4 values, but 1 was provided

USAGE:
    tk [OPTIONS] [--] [tasks]...

For more information try --help

```

Or with that:
```
❯ tk --init=jbang bang.java 
error: The argument '--init <shell> <make> <jbang> <just>' requires 4 values, but 1 was provided

USAGE:
    tk [OPTIONS] [--] [tasks]...

For more information try --help

```

Version:
```
❯ tk -V
tk 0.6.1
```

**Comment by nandorholozsnyak** [comment]
Just upgraded, still happening.
```
❯ tk -V                    
tk 0.6.2
```

**Comment by linux-china** [comment]
Hi man, sorry for delay feedback.  Fixed in 0.6.3, and try `tk --init=jbang`  to create demo jbang-catalog.json

**Comment by nandorholozsnyak** [comment]
Hi.
Thank you.
